formidable/app.py

```diff
@@ -21,3 +25,9 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
+
+        # Deprecation warning for django 1.8 and 1.9
+        if StrictVersion(django.get_version()) <= StrictVersion("1.9"):
```

Comment:
here, you only warn for Django 1.8.

The correct line:

```python
if StrictVersion(django.get_version()) < StrictVersion("1.10"):
```

Tested using this:

```python
        # Deprecation warning for django 1.8 and 1.9
        if StrictVersion(django.get_version()) < StrictVersion("1.10"):
            raise Exception("MEUH")
            warnings.warn("Support for Django 1.8 and 1.9 is about to be "
                          "dropped. Please update to a newer version.",
                          DeprecationWarning)

```

If you run your former condition, you see that the tests fail only for `django18-*`.

Reply:
I'm running the demo site with Django 1.9 and I do have the warning raised, that's what tricked me. Indeed with Django 1.9.X it won't work, as I would have found launching tox with 1.9. I'm disappointed with myself :thinking: